lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -380,6 +381,24 @@ class ReportUIFeatures {
     this._document.body.removeChild(a);
     setTimeout(_ => URL.revokeObjectURL(href), 500);
   }
+
+  /**
+   * Register listeners to toggle dark theme via clicking on LH logo or pressing 'd'.
+   * @private
+   */
+  _registerDarkThemeToggles() {
+    // TODO(#8185) - Persist dark mode state.
+    this._document.addEventListener('keyup', e => {
```

Comment:
should really combine this with `printShortCutDetect` in a single key handler so they don't keep multiplying

Reply:
a generic "onKeyDown" handler would separate related code, which seems worse than an extra event handler.

also, this is keyup but that is keydown (nbd it'd just repeatedly remove a class if you decide to hold Ctrl + P)

Reply:
the related code is really just
```js
_toggleDarkMode() {
  this._document.body.classList.toggle('dark');
}
```
 :)

agreed keyup is the way to go, `printShortCutDetect` should be using that.

Reply:
We don't have to if you feel strongly about keeping these together. Three key handlers is just starting to feel messy to me personally.

Reply:
done. I actually found that `collapseAllDetails` was erroring (at least since 4.3) b/c it was binding (Cmd + P -> cancel print).